dtfs-central-api/src/v1/controllers/utilisation-report-service/upload-utilisation-report.controller.js

```diff
@@ -4,19 +4,19 @@ const {
   validateUtilisationReportData,
   validateMonth,
   validateYear,
-  validateFilePath,
+  validateFileInfo,
   validateBankId,
 } = require('../../validation/utilisation-report-service/utilisation-report-validator');
 const db = require('../../../drivers/db-client');
 
-const validatePayload = (reportData, month, year, filePath, bankId) => {
+const validatePayload = (reportData, month, year, fileInfo, bankId) => {
   const validationErrors = validateUtilisationReportData(reportData);
   const monthValidationError = validateMonth(month);
   if (monthValidationError) validationErrors.push(monthValidationError);
   const yearValidationError = validateYear(year);
   if (yearValidationError) validationErrors.push(yearValidationError);
-  const filePathValidationError = validateFilePath(filePath);
-  if (filePathValidationError) validationErrors.push(filePathValidationError);
+  const fileInfoValidationError = validateFileInfo(fileInfo);
+  if (fileInfoValidationError) validationErrors.push(...fileInfoValidationError);
```

Comment:
I think this should be `if (fileInfoValidationErrors.length)` as an empty array is truthy.